www/server/tools/componentStatusHelper.js

```diff
@@ -0,0 +1,80 @@
+import config from '../config';
+import https from 'https';
+import fs from 'fs';
+import path from 'path';
+import { kebabCase } from 'lodash';
+
+const PATH_JSON = path.resolve(__dirname, '../componentStatus');
+
+const save = (path, fileName,data) => {
+  fs.writeFile(path + '/' + fileName,data,{
+    encoding:'utf8',
+    mode:0o666,
+    flag:'w'
+  }, function(err) {
+    if (err) throw err;
+  });
+};
+
+const getReactComponentList = str => {
+  let res = str.match(/\'(.+?)\'/g);
+  return res.map(function(item){
+    return kebabCase(item.replace(/'/g, ''));
+  });
+};
+
+const getAngularComponentList = str => {
+  let res = str.match(/\'\.\/(.+?)\//g);
+  return res.map(function(item){
+    return item.replace(/'\.\/|\//g, '');
+  });
+};
+
+const downloadFile = (fileName, callback) => {
```

Comment:
Instead of creating a file on the server, it would be more persistent to push this info to the component database. Then access it through the api endpoint. Another option would be to just create it at build time in the client and not use the server api at all.

Reply:
Store the info in database is good, but it will need additional efforts to keep it update to date constantly. And I don't have the admin account of wp.momentum.design.
Create it at build time also has the issue how to keep it update to date.

Reply:
There would be no additional effort to keep updated at build time, since we build and publish the site every time we publish the libraries.

Reply:
I see. I'll move it to the build part.

Reply:
Component status JSONs are created at build time now.